### PR TITLE
SALTO-2226/omit specific permissions from permission schemes

### PR DESCRIPTION
**Pull request by guyhamo**
omit "VIEW_PROJECTS" and "VIEW_ISSUES" permissions from permission schemes since they are invisible in the ui and cant be deployed.

---

_Additional context for reviewer_
going to suppress notifications in saas aswell

---
_Release Notes_: 
Removed redundant values from permission scheme instances to fix the deployment of permission schemes

---
_User Notifications_: 
removed "VIEW_PROJECTS" and "VIEW_ISSUES" permissions from permission scheme NaCL files.

### CODE REVIEW

File: packages/jira-adapter/src/filters/forbidden_permission_schemes.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  'VIEW_ISSUES',
+]
+/**
+ * Remove unsupported permissions from permission schemes
+ */
+const filter: FilterCreator = () => ({
+  onFetch: async (elements: Element[]) => {
+    elements.filter(isPermissionScheme).filter(isInstanceElement).forEach(element => {
+      _.remove(element.value.permissions,
+        (p:{ permission: string }) => UnsupportedPermissionSchemes.includes(p.permission))
```
Comment (alonstern) [inline packages/jira-adapter/src/filters/forbidden_permission_schemes.ts:34]: super nit: add space between the `p:` and the `{` `p: { permission: string }`

**Review by alonstern — APPROVED** [review]
👌👌
Could you also add to the release notes something like:
`Removed redundant values from permission scheme instances to fix the deployment of permission schemes`

**Comment by guyhamo** [comment]
> 👌👌 Could you also add to the release notes something like: `Removed redundant values from permission scheme instances to fix the deployment of permission schemes`
khaby

**Comment by ori-moisis** [comment]
@guyhamo in the future please include which adapter is this related to in the release notes / user notifications section

**Pull request merged**